packages/lib/freeEmailDomainCheck/checkIfFreeEmailDomain.ts

```diff
@@ -0,0 +1,43 @@
+import { freeEmailDomainsThatStartWithANumberObject, freeEmailDomains } from "./freeEmailDomains";
+
+export const checkIfFreeEmailDomain = (email: string) => {
+  const emailDomain = email.split("@")?.[1];
```

Comment:
I am not sure if email could be in capital here but I think we should lowercase it just to be sure.